app/appConfig.ts

```diff
@@ -12,19 +12,28 @@ const isEnvironment = (env: ReleaseChannel): boolean =>
 const devConfig: config.Config = {
   apiEndpoint: "https://uat-proxy.idem.com.au/", // change to localhost if running locally for example "http://192.168.1.117:3000/",
   userName: "myusername",
-  password: "password123"
+  password: "password123",
```

Comment:
Should we be consistent and just keep all passwords as secrets regardless of env?

Reply:
dev built are built locally on dev machine without getting injected by expo secret